src/libs/freetype/lv_freetype_image.c

```diff
@@ -176,7 +127,7 @@ static void freetype_image_release_cb(const lv_font_t * font, lv_font_glyph_dsc_
 {
     LV_ASSERT_NULL(font);
     lv_freetype_font_dsc_t * dsc = (lv_freetype_font_dsc_t *)font->dsc;
-    lv_cache_release(dsc->cache_node->image_cache, g_dsc->entry, NULL);
+    lv_cache_release(dsc->cache_node->draw_data_cache, g_dsc->entry, NULL);
```

Comment:
I personally find it hard to understand these words `dsc->cache_node->draw_data_cache`. I would like to hear from others opinions.

Reply:
Cache of data to be drawn